corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -128,6 +128,53 @@
   </div>
 </script>
 
+<script type="text/template" id="case-tile-grouped-view-item-template">
+  <% if (isMultiSelect) { %>
+    <div class="select-row-checkbox-div">
+      <input type="checkbox" class="select-row-checkbox"/>
+    </div>
+  <% } %>
+
+  <div class = "group-data">
+    <div class="<%- prefix %>-cell-grid-style">
+      <% _.each(headerData, function(datum, index) { %>
+        <div class="<%- prefix %>-grid-style-<%- index %> box" >
+          <% if (styles[index].displayFormat === 'Image') {
+          if(resolveUri(datum)) { %>
+            <img class="module-icon" style="max-width:100%; max-height:100%;" src="<%- resolveUri(datum) %>"/>
+          <% } %>
+          <% } else if(styles[index].widthHint === 0) { %>
+            <div style="display:none;"><%- datum %></div>
+          <% } else { %>
+            <div class="webapp-markdown-output"><%= renderMarkdown(datum) %></div>
+          <% } %>
+        </div>
+      <% }); %>
```

Comment:
Aside: It would be great if we didn't have to repeat all of this template. I was hoping you could do nested templates but it seems that would require a lot more work.

Reply:
Agreed. I don't think it needs to be part of this PR but do think it's worth a little investigation. Possibly a similar approach to `buildCaseTileStyles`, where the view's `initialize` function calls template functions, passing them only the data they need, and then injects the returned HTML strings into containers in the "parent" template?

Reply:
I think this portion can be pulled out as a template since `case-tile-view-item-template` and `case-tile-grouped-view-item-template` both header and rows all essentially use the same index,datum loop pattern. 
Then any specific html class and be injected via the view?
https://github.com/dimagi/commcare-hq/blob/1b35ebf340d2b451e5dcfb5b4b4e5dc3420ee0d1/corehq/apps/cloudcare/templates/formplayer/case_list.html#L114-L128